File: cpp/funkcje.cpp
/*
 * szablon.cpp
 */


#include <iostream>

using namespace std;

int suma(int a, int b){
    
    return a + b;
    
    }
    
int roznica(int a, int b){
    
    return a - b;
    
    }
    
int iloczyn(int a, int b){
    
    return a * b;
    
    }
    
int iloraz(int a, int b){
    
    return a / b;
    
    }
int main(int argc, char **argv)
{ /* DRY - Don't repat yourself */
	
    int a, b;
    
    a = b = 0;
    
    cout << "Podaj liczbę: ";
    cin >> a;
    
    cout << endl << "Podaj drugą liczbę: ";
    cin >> b;
    
    cout << a << "," << b;
    
    cout << endl << "suma: "; 
    cout << suma(a, b);
    
    cout << endl << "różnica: ";
    cout << roznica(a, b);
    
    cout << endl << "iloczyn: ";
    cout << iloczyn(a, b);
    
    cout << endl << "iloraz: ";
    cout << iloraz(a, b);
    
    
    
	return 0;
}

File: python/funkcje.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  wejscie_wyjscie.py
#  
# zasięg zmiennych: lokalny(ograniczony funkcją)


def suma(a, b):
    return a + b

def roznica(a, b):
    return a - b
    
def iloczyn(a, b):
    return a * b
    
def iloraz(a, b):
    return a / b

def main(args):
    a = int(input("Podaj pierszą liczbę: "))
    print(a)
    b = int(input("Podaj drugą liczbę: "))
    print(b)
    
    print("Suma: ", suma(a, b))
    print("Różnica: ", roznica(a, b))
    print("Iloczyn: ", iloczyn(a, b))
    print("Iloraz: ", iloraz(a, b))
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
